cmake/Platform/Core/LibraryFinder.cmake

```diff
@@ -144,6 +144,6 @@ function(find_arduino_libraries VAR_NAME SRCS ARDLIBS)
     endif ()
 
     _REMOVE_BLACKLISTED_LIBRARIES("${ARDUINO_LIBS}" FILTERED_LIBRARIES)
-    set(${VAR_NAME} ${FILTERED_LIBRARIES} PARENT_SCOPE)
+    set(${VAR_NAME} "${FILTERED_LIBRARIES}" PARENT_SCOPE)
```

Comment:
It is always better to quote lists when passing them as they might contain spaces.